subprojects/base-services/src/main/java/org/gradle/internal/progress/BuildOperationDescriptor.java

```diff
@@ -21,21 +21,27 @@
 /**
  * Meta-data about a build operation.
  */
-public class BuildOperationDetails {
-    private final BuildOperationExecutor.Operation parent;
+public class BuildOperationDescriptor {
+    private final Long id;
```

Comment:
Maybe it would be better then to create a new class `BuildOperationIdentifier` and use that. `OperationIdentifier` is currently used for the ids of other _kinds_ of operations as well. So we would still allow _compare apples to oranges_ situation.